Description: Don't register a handler for SIGSEGV. We want Apport to catch these crashes so that users can submit useful crash reports.
Author: ?

Index: gnome-session-2.27.4/gnome-session/gdm-signal-handler.c
===================================================================
--- gnome-session-2.27.4.orig/gnome-session/gdm-signal-handler.c	2009-07-24 20:14:23.000000000 +0100
+++ gnome-session-2.27.4/gnome-session/gdm-signal-handler.c	2009-07-24 20:17:41.000000000 +0100
@@ -348,7 +348,7 @@
 
         gdm_signal_handler_add (handler, SIGILL, NULL, NULL);
         gdm_signal_handler_add (handler, SIGBUS, NULL, NULL);
-        gdm_signal_handler_add (handler, SIGSEGV, NULL, NULL);
+        /* gdm_signal_handler_add (handler, SIGSEGV, NULL, NULL); */
         gdm_signal_handler_add (handler, SIGABRT, NULL, NULL);
         gdm_signal_handler_add (handler, SIGTRAP, NULL, NULL);
 }
